.github/workflows/build_cmake.yml

```diff
@@ -1,113 +1,134 @@
 name: CMake Build
-
 on: [push, pull_request]
 
 jobs:
   linux:
     name: ${{ matrix.config.name }}
     runs-on: ubuntu-latest
+    env:
+      build_type: "Release"
```

Comment:
Can you add a prefix like `inexor` as thos names are actually added as environment variables and my override others.
(You can use them with `$build_type` too!)